From 3bd9ff501328e1c5e88721ef86164ad7415c9327 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 10 Jan 2017 00:40:50 +0100
Subject: [PATCH 2/7] Use <var> instead of angle bracket entities

This gets rid of these entities, which otherwise get lost in the DocBook
conversion, and also switches to use the more correct markup anyway.
---
 policy.sgml | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 9810090..afa05a6 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -3391,8 +3391,8 @@ Package: libc6
 
 	  <p>
 <example>
-	Description: &lt;single line synopsis&gt;
-	 &lt;extended description over several lines&gt;
+	Description: <var>single line synopsis</var>
+	 <var>extended description over several lines</var>
 </example>
 	  </p>
 
@@ -7106,10 +7106,10 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
 	      </item>
 	      <item>
 		<p>
-                  The requirement for <file>/usr/local/lib&lt;qual&gt;</file>
-                  to exist if <file>/lib&lt;qual&gt;</file> or
-                  <file>/usr/lib&lt;qual&gt;</file> exists (where 
-                  <file>lib&lt;qual&gt;</file> is a variant of
+                  The requirement for <file>/usr/local/lib<var>qual</var></file>
+                  to exist if <file>/lib<var>qual</var></file> or
+                  <file>/usr/lib<var>qual</var></file> exists (where
+                  <file>lib<var>qual</var></file> is a variant of
                   <file>lib</file> such as <file>lib32</file> or
                   <file>lib64</file>) is removed.
                   </p>
@@ -7802,11 +7802,11 @@ test -f <var>program-executed-later-in-script</var> || exit 0
 	    <p>
 	      Most packages will simply need to change:
 	      <example compact="compact">
-/etc/init.d/&lt;package&gt; &lt;action&gt;
+/etc/init.d/<var>package</var> <var>action</var>
 	      </example> in their <prgn>postinst</prgn>
 	      and <prgn>prerm</prgn> scripts to:
 	      <example compact="compact">
-invoke-rc.d <var>package</var> &lt;action&gt;
+invoke-rc.d <var>package</var> <var>action</var>
 	      </example>
 	    </p>
 
@@ -9962,7 +9962,7 @@ http://localhost/cgi-bin/.../<var>cgi-bin-name</var>
                 may be referred to through an alias <tt>/images/</tt>
                 as
                 <example>
-                  http://localhost/images/&lt;package&gt;/&lt;filename&gt;     
+                  http://localhost/images/<var>package</var>/<var>filename</var>
                 </example>
                 
               </p>
-- 
2.11.0

